src/components/Postcard-Preview/_Postcard-Preview.scss

```diff
@@ -0,0 +1,86 @@
+.preview {
+  @include display-flex;
+  @include paragraph-font
+
+  button {
+    @include button()
+  }
+
+  .front {
+    border: 1px solid $ink-blue;
+    margin: 1rem;
+    padding: .5rem .35rem .5rem;
+    width: 95%;
+    height: 40vh;
+    object-fit: cover;
+    overflow: hidden;
+    background: $off-white;
```

Comment:
Was this color used to differentiate between our "white" and the postcard itself?